frontend/test/services/transaction.test.js

```diff
@@ -41,107 +36,18 @@ describe('Transaction Functions', () => {
     };
 
     connect.mockResolvedValue(mockStarknet);
-
-    process.env.REACT_APP_BACKEND_URL = mockBackendUrl;
-  });
-
-  afterEach(() => {
-    delete process.env.REACT_APP_BACKEND_URL;
-  });
-
-  describe('sendTransaction', () => {
-    const validLoopLiquidityData = {
-      pool_key: '0x123',
-      deposit_data: {
-        token: '0x456',
-        amount: '1000',
-      },
-    };
-
-    it('should successfully send a transaction', async () => {
-      const result = await sendTransaction(validLoopLiquidityData, mockContractAddress);
-
-      expect(connect).toHaveBeenCalled();
-      expect(result).toEqual({
-        loopTransaction: mockTransactionHash,
-      });
-    });
-
-    it('should throw error if wallet is not connected', async () => {
-      connect.mockResolvedValueOnce({ isConnected: false });
-
-      await expect(sendTransaction(validLoopLiquidityData, mockContractAddress)).rejects.toThrow(
-        'Wallet not connected'
-      );
-    });
-
-    it('should throw error if loop_liquidity_data is invalid', async () => {
-      const invalidData = { deposit_data: { token: '0x456' } };
-
-      await expect(sendTransaction(invalidData, mockContractAddress)).rejects.toThrow(
-        'Missing or invalid loop_liquidity_data fields'
-      );
-    });
-
-    it('should handle transaction errors correctly', async () => {
-      const mockError = new Error('Transaction failed');
-      connect.mockResolvedValueOnce({
-        isConnected: true,
-        account: {
-          execute: jest.fn().mockRejectedValue(mockError),
-        },
-      });
-
-      console.error = jest.fn();
-
-      await expect(sendTransaction(validLoopLiquidityData, mockContractAddress)).rejects.toThrow('Transaction failed');
-
-      expect(console.error).toHaveBeenCalledWith('Error sending transaction:', expect.any(Error));
-    });
   });
 
-  describe('closePosition', () => {
```

Comment:
you must not deleted tests. it should be the same amount as we have in main now, I see 8 less

Reply:
unresolved, you have function closePosition, but I don't see a test for it.